### ISSUE TITLE
Retrieve "My Tournaments" 

### ISSUE THREAD

**Issue by alex-stetson** [TournamentBuzz/main#78]
## Story/Task Details
- Input: JWT of current user
- Output: Array of tournament objects the current user is associated with, potentially along with what their association is with each particular tournament is (participant, organizer, etc.)

**Comment by dli357** [comment]
## API Endpoints
| Endpoint | Description | Request Method | Expected Input | Expected Output |
|-----------|--------------|--------------------|------------------|--------------------|
| `/user/tournaments` | Returns a list of tournaments that the user is in | `GET` | `Valid JWT` | `{tournaments: [<tournamentObject>]}` |

```{
id: <number>,
creator: <string>,
description: <string>,
teamEvent: <boolean>,
location: <string>,
scoringType: <enum('Points')>,
tournamentName: <string>, 
tournamentType: <enum('Single Elim', 'Double Elim', 'Round-robin')>,
entryCost: <int>,
maxParticipants: <int>,
startDate: <dateTime>,
endDate: <dateTime>
}```

**Comment by dli357** [comment]
Implemented with commit 08d6ba0e43a99b57e536ba761891fc581e2b3d5f.
